src/main/scala/Column.scala

```diff
@@ -286,19 +285,37 @@ object Column {
       if (datatype.path("length").isMissingNode) None
       else Some(datatype.get("length").asText().toInt)
 
+    val minInclusive: Option[BigDecimal] =
+      if (datatype.path("minInclusive").isMissingNode) None
+      else Some(BigDecimal(datatype.get("minInclusive").asText()))
```

Comment:
Is there some way we can get the `BigDecimal` out of the `JsonNode` without converting it to text?

Reply:
I don't think we can. So the other functions available on a fetched properties are `asInt()` `asLong()` `asDouble()` 
If we don't call any function after trying to get a property it will be just a `jsonNode`

Reply:
ok, cool